examples/multi_modal/dataloader.py

```diff
@@ -0,0 +1,190 @@
+"""Lightning dataloader for text and image data."""
+
+import logging
+import os
+from typing import Any, Union
+
+import joblib
+import lightning as pl
+import numpy as np
+import torch
+from lightning import seed_everything
+from litdata import StreamingDataLoader, StreamingDataset
+from torch.utils.data import DataLoader
+from torchvision import transforms
+from transformers import BertTokenizerFast
+
+from examples.multi_modal.config import HYPERPARAMETERS
+
+pil_transform = transforms.Compose([transforms.PILToTensor()])
+
+
+logger = logging.getLogger()
+
+
+os.environ["TOKENIZERS_PARALLELISM"] = "true"
+seed = seed_everything(21, workers=True)
+
+
+class EncoderAndTokenizer:
+    def __init__(self):
+        self.hyperparameters = HYPERPARAMETERS
```

Comment:
this is passed as an argument everywhere else. I think it would be cleaner not to use this from the global

Reply:
No